python/tests/test_aac/plugins/gen_plugin/my_plugin.aac

```diff
@@ -21,7 +21,53 @@ plugin:
         - name: Command test
           scenarios:
             - name: Simple Command Test
-              given: 
+              given:
+                - An AaC file containing schemas with no extra fields.
+              when:
+                - The AaC check command is run on the schema.
+              then:
+                - The check commands provides the output 'All AaC constraint checks were successful.'
+    - name: test-command-two
+      help_text: |
+        A second test command used to test the gen-plugin command behavior.
+      run_before:
+        - plugin: Check AaC
+          command: check
+      run_after:
+        - plugin: Generate
+          command: generate
+      input:
+        - name: aac-plugin-file
+          type: file
+          description: The path to the architecture file with the plugin definition.
+      acceptance:
+        - name: Command test
```

Comment:
Not sure you will get expected results for testing by having them named the same every time. 
```suggestion
        - name: Command test two
```